sauna/main.py

```diff
@@ -16,6 +18,7 @@
 Available commands:
 """
 import logging
+import logging.config
```

Comment:
Yes, otherwise you cannot access the dictConfig function.

```
>>> import logging
>>> logging.config.dictConfig({'version': 1})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'config'
>>> import logging.config
>>> logging.config.dictConfig({'version': 1})
>>> 
```